example-app/src/app.ts

```diff
@@ -48,7 +51,14 @@ async function main() {
   app.use(logging(rootLogger)); // <-- logging middleware
   app.use(missedVaas(app, { namespace: "simple", logger: rootLogger }));
   app.use(providers());
-  // app.use(wallets({ logger: rootLogger, namespace: "simple", privateKeys })); // <-- you need a valid private key to turn on this middleware
+  app.use(
+    wallets(env, {
+      logger: rootLogger,
```

Comment:
meta idea: could these common config points be settable at the app level? There could be usecases where these make sense to be set per-middleware, but I imagine most times this is just boilerplate